src/main/java/bio/terra/cli/command/workspace/Clone.java

```diff
@@ -21,7 +21,7 @@
 /** This corresponds to the third-level "terra workspace clone" command. */
 @Command(name = "clone", description = "Clone an existing workspace.")
 public class Clone extends BaseCommand {
-  @CommandLine.Option(names = "--id", required = true, description = "Source workspace ID")
+  @CommandLine.Option(names = "--new-id", required = true, description = "ID for new workspace")
```

Comment:
Double check this is what happens. I.e. that it applies to the destination and not the source. (We have commands that take in a `--workspace` argument to set the active workspace, and I'm worried this might have been one of those and we no longer have the ability to do that).

Reply:
Yes. after I tested. the `--new-id` represent the new id. and the `--workspace` represent the old(current active) workspace id. See the link for testing.
https://paste.googleplex.com/4896313884803072